Src/Workspaces/Core/Portable/Simplification/AbstractSimplificationService.cs

```diff
@@ -231,7 +231,7 @@ protected virtual SyntaxNode TransformReducedNode(SyntaxNode reducedNode, Syntax
             }, cancellationToken);
             }
 
-            Task.WaitAll(simplifyTasks, cancellationToken);
```

Comment:
is WithCancellation needed? simplifyTasks is already hot. if there is any cancellation, it will throw.